sdk/python/tests/test_feature_set.py

```diff
@@ -167,3 +170,94 @@ def test_add_features_from_df_success(
         )
         assert len(my_feature_set.features) == feature_count
         assert len(my_feature_set.entities) == entity_count
+
+    def test_import_tfx_schema(self):
+        tests_folder = pathlib.Path(__file__).parent
```

Comment:
Doesn't this depend on where pytest is launched from?

Reply:
Oh `pathlib.Path(__file__)` will give the correct relative path to `test_feature_set.py`. If pytest is called from different different context, the relative path will change to the correct location too.